downstream/modules/hub/proc-add-group-to-container-repo.adoc

```diff
@@ -13,7 +13,8 @@ Provide access to your container repository for users who need to work the image
 
 . Navigate to menu:Execution Environments[].
```

Comment:
This should be updated based on UI changes and I believe it should be as follows but should be verified with SMEs:
From the navigation panel, select menu:Execution Environments[Remote registries].
Click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Edit*.